src/main/cpp/controllers/DrivetrainController.cpp

```diff
@@ -141,6 +157,21 @@ Eigen::Vector<double, 2> DrivetrainController::Calculate(
     return m_u;
 }
 
+Eigen::Vector<double, 2> DrivetrainController::CalculateTurnInPlace(
+    const Eigen::Vector<double, 3>& x) {
+    m_u = Eigen::Vector<double, 2>::Zero();
```

Comment:
This a dead write because m_u is completely overwritten later. This line can be removed.